src/components/timers/XY.js

```diff
@@ -58,7 +58,7 @@ class XY extends React.Component {
     const displayTimes = [
       <DisplayTime
         seconds={workSecs}
-        key="1"
+        key="1" 
```

Comment:
small nit, but I like to make these keys more descriptive when I can 
```suggestion
        key="display-work-secs"
```